automated install/basic-install.sh

```diff
@@ -702,6 +716,30 @@ find_IPv4_information() {
     IPV4_ADDRESS=$(ip -oneline -family inet address show | grep "${IPv4bare}/" | awk '{print $4}' | awk 'END {print}')
 }
 
+confirm_ipv6_only() {
+    # Confirm from user before IPv6 only install,
+    # unless PI_HOLE_IPV6_ONLY environment variable is true,
+
+    dialog --no-shadow --output-fd 1 \
```

Comment:
This dialog selection defaults to no ('cancel') - is this intended?

Reply:
It was intended, but that was when the connectivity issue on non nat64 systems was at issue. 
Changed to default yes in new commit.